src/profile.command.ts

```diff
@@ -45,6 +49,36 @@ export abstract class ProfileCommand extends Command {
         }
     }
 
+    public getAccount(options: ProfileOptions): Account {
+        const profile = this.getProfile(options);
+
+        const password = options.password || readlineSync.question('Enter your wallet password: ');
+        new PasswordValidator().validate(password);
+        const passwordObject = new Password(password);
+
+        if (!profile.isPasswordValid(passwordObject)) {
+            throw new ExpectedError('The password you provided does not match your account password');
+        }
+
+        return profile.simpleWallet.open(passwordObject);
+    }
+
+    public getAccountAndProfile(options: ProfileOptions): {account: Account, profile: Profile} {
+        const profile = this.getProfile(options);
+        const account = this.getAccount(options);
+        return { account, profile };
+    }
+
+    public getAddress(options: ProfileOptions): Address {
+        const profile = this.getProfile(options);
+
+        return Address.createFromRawAddress(
```

Comment:
I would prefer to put this line in every command needed instead of abstracting it in a function. It will be more clear what's being done underneath, it's only one line of code, and we save calling this.getProfile(options) more than once per command.

Reply:
are you sure about this one? 
The address is needed in all the queries under `commands/account`, in the same fashion as the account is needed for transactions commands. I think it is good design to do it like that